packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -980,6 +1075,22 @@ export default class PciInstancesAddController {
           );
         },
       );
+
+      if (
+        this.isLocalPrivateModeLocalZone &&
+        this.isCreatingNewPrivateNetwork &&
+        this.privateNetworkName
+      ) {
+        this.instance.networks = [
+          ...(this.instance.networks || []),
+          {
+            networkId: this.availableLocalPrivateNetworks.find((n) => {
+              return n.name === this.privateNetworkName;
```

Comment:
Use fullname or destructuring. But no short names.
```suggestion
            networkId: this.availableLocalPrivateNetworks.find(({ name }) => {
              return name === this.privateNetworkName;
```